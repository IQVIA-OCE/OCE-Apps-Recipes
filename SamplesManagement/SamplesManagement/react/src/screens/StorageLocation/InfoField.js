import React from 'react';
import { StyleSheet, View } from 'react-native';
import { Text } from 'apollo-react-native';

const InfoField = ({ label, text, style }) => {
  return (
    <View style={[styles.root, style]}>
      <Text style={styles.label}>{label}</Text>
      <Text style={styles.text}>{text}</Text>
    </View>
  );
};

const styles = StyleSheet.create({
  root: {
    width: '100%',
    marginBottom: 15,
  },
  label: {
    marginBottom: 15,
  },
  text: {
    fontSize: 16,
  },
});

export default InfoField;
