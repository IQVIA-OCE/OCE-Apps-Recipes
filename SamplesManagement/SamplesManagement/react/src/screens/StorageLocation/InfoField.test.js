import React from 'react';
import renderer from 'react-test-renderer';
import InfoField from './InfoField';

describe('StorageLocation InfoField', () => {
  it('Should render InfoField component', () => {
    const tree = renderer.create(
      <InfoField
        label="label"
        text="text"
        style={{}}
      />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
