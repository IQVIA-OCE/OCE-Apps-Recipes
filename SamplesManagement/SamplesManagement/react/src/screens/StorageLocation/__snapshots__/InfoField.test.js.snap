// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StorageLocation InfoField Should render InfoField component 1`] = `
<View
  style={
    Array [
      Object {
        "marginBottom": 15,
        "width": "100%",
      },
      Object {},
    ]
  }
>
  <Text
    style={
      Object {
        "marginBottom": 15,
      }
    }
  >
    label
  </Text>
  <Text
    style={
      Object {
        "fontSize": 16,
      }
    }
  >
    text
  </Text>
</View>
`;
